lib/ansible/modules/storage/netapp/netapp_e_storagepool.py

```diff
@@ -4,808 +4,907 @@
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import absolute_import, division, print_function
-__metaclass__ = type
-
 
-ANSIBLE_METADATA = {'metadata_version': '1.1',
-                    'status': ['preview'],
-                    'supported_by': 'community'}
+__metaclass__ = type
 
+ANSIBLE_METADATA = {"metadata_version": "1.1",
+                    "status": ["preview"],
+                    "supported_by": "community"}
 
-DOCUMENTATION = '''
+DOCUMENTATION = """
 ---
 module: netapp_e_storagepool
-short_description: NetApp E-Series manage disk groups and disk pools
-version_added: '2.2'
-description:
-    - Create or remove disk groups and disk pools for NetApp E-series storage arrays.
+short_description: NetApp E-Series manage volume groups and disk pools
+description: Create or remove volume groups and disk pools for NetApp E-series storage arrays.
+version_added: "2.2"
```

Comment:
```suggestion
version_added: '2.2'
```

Reply:
Should version_added values always be wrapped in single quotes?